docs/preview.md

```diff
@@ -3,8 +3,9 @@
 Ruff includes an opt-in preview mode to provide an opportunity for community feedback and increase confidence that
 changes are a net-benefit before enabling them for everyone.
 
-Preview mode enables a collection of newer lint rules, fixes, and formatter style changes that are
-considered experimental or unstable,.
+Preview mode enables a collection of new lint rules, fixes, formatter style and other changes that are considered experimental or unstable.
+
+Simply enabling preview mode does not imply that all the new preview features/changes will be turned on. Some, like the rules that are in preview, need to be turned on explicitly (see the [rules section](#using-rules-that-are-in-preview). Others, like the formatter style or default values (for example `output-format`) will be turned on by simply enabling the preview mode.
```

Comment:
Nit: I would avoid explicitly naming `output-format` here. We otherwise have to come up with a new preview feature everytime we remove the feature mentioned in the documentation.

Reply:
I hesitated when writing it, better keep it general indeed.
Removed in b0a9db688.